static/src/stylesheets/amp/_hosted.scss

```diff
@@ -0,0 +1,848 @@
+$header-height: 48px;
+$thumb-width: 90px;
+$badge-width: 80px;
+
+.hosted-page {
+    @include f-textSans;
+}
+
+.hosted__glogo {
+    display: inline-block;
+    padding: $gs-baseline/3 $gs-gutter/2;
+    background-color: $guardian-brand;
+    text-align: right;
+}
+
+.hosted__guardian-logo__svg {
+    fill: #ffffff;
+    width: 115px;
```

Comment:
this is the logo:

![image](https://cloud.githubusercontent.com/assets/6290008/20890709/c1938d52-baff-11e6-818d-2788021e4493.png)

The svg has width/height attributes which make it too big for the space. I'm not sure how to shrink it other than explicitly sizing it like this!